src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* `Enrich` callback is now split into 3 different callbacks based on the object
```

Comment:
we'll need a bigger warning about the breaking change.

Reply:
See this too : https://github.com/open-telemetry/opentelemetry-dotnet/pull/3749#discussion_r991596864
@alanwest We need your help to write a good changelog. (and we can borrow this for all other similar PRs)